### PR TITLE
Add a AS/IJ plugin doc

### PR DESCRIPTION
**Pull request by BoD**

### CODE REVIEW

File: docs/source/config.json [hunk 1]
```diff
@@ -61,11 +61,12 @@
...
     },
     "Development & Testing": {
-      "Overview": "/testing/overview",
+      "Testing overview": "/testing/overview",
       "Mocking HTTP responses": "/testing/mocking-http-responses",
       "Mocking GraphQL responses": "/testing/mocking-graphql-responses",
       "Data builders": "/testing/data-builders",
-      "UI Tests": "/testing/ui-tests"
+      "UI Tests": "/testing/ui-tests",
+      "Android Studio plugin": "/testing/android-studio-plugin"
```
Comment (martinbonnin) [inline docs/source/config.json:69]: I would move it up before testing in the list because you should set this up before you start testing. Also update the "Get Started" to link to this page?

File: docs/source/testing/android-studio-plugin.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Installation
+
+The plugin is available on the JetBrains plugin repository. To install it from the IDE, go to
+<kbd>Settings</kbd> | <kbd>Plugins</kbd> | <kbd>Marketplace</kbd> | search for "Apollo GraphQL" | <kbd>Install</kbd>
+
+The plugin is supported on:
+
+- IntelliJ 2022.2 and above
+- Android Studio 2022.2 (Flamingo) and above
```
Comment (martinbonnin) [inline docs/source/testing/android-studio-plugin.mdx:15]: This is a recipe for stale information as this is changing fast. I would rephrase with something like so at least to have some failsafe:

```suggestion
At the time of writing, the plugin is supported on:

- IntelliJ 2022.2 and above
- Android Studio 2022.2 (Flamingo) and above

Please check the [plugin page](https://plugins.jetbrains.com/plugin/20645-apollo-graphql) for up to date information.
```

File: docs/source/testing/android-studio-plugin.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+### Integration with the GraphQL IntelliJ Plugin
+
+The plugin depends on and integrates with the [GraphQL IntelliJ Plugin](https://plugins.jetbrains.com/plugin/8097-js-graphql).
+The file structure of the Apollo project is automatically contributed, so there is no need to create a `graphql.config.yml` / `.graphqlconfig` file.
+
+### Navigation
+
+#### Kotlin → GraphQL
+
+When editing Kotlin code, <img src="images/gutter-icon.png" alt="Gutter icon" width="16" valign="top"/> icons will show next to references to generated operation and fragment classes.
```
Comment (martinbonnin) [inline docs/source/testing/android-studio-plugin.mdx:35]: Inline images do not work :(. I'm no css expert but maybe a `display: inline-block;` would work here?

Screenshot 2023-07-25 at 13 49 30

File: docs/source/testing/android-studio-plugin.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+<img src="images/high-latency-field-editor.png" width="400" alt="High latency in editor"/>
+
+By default, fields with a latency above 100ms will be highlighted. You can change this threshold in <kbd>Settings</kbd> | <kbd>Editor</kbd> | <kbd>Inspections</kbd> | <kbd>GraphQL</kbd> | <kbd>Apollo GraphOS</kbd> | <kbd>High latency field</kbd>.
+
+A quick fix is available to encapsulate the field in a fragment with the [`@defer`](../fetching/defer) directive.
+
+For the plugin to be able to fetch the metrics, you need to configure your project with its [Apollo GraphOS API key](https://www.apollographql.com/docs/graphos/api-keys/). To do this, go to the plugin's settings: <kbd>Settings</kbd> | <kbd>Languages & Frameworks</kbd> | <kbd>GraphQL</kbd> | <kbd>Apollo Kotlin</kbd>. Here you can find the <kbd>API keys</kbd> table where you can add:
+- API key: should start with `service:` or `user:`
+- Graph ID: can be found on your graph page in Apollo Studio
```
Comment (martinbonnin) [inline docs/source/testing/android-studio-plugin.mdx:78]: I think we're able to infer the Graph ID from the API key?

```suggestion
- API key: should start with `service:` or `user:`
- Graph ID: can be found on your graph page in Apollo Studio or is filled automatically if you provide a `service:` API key
```

**Review by martinbonnin — APPROVED** [review]
A few comments but LGTM 🚀 

Reading this I was wondering how much of an effort it would be to make a small "tour" screencast video. That would be perfect to showcase the different features here.

ping @Meschreiber, @trevor-scheer do we have that capability in the docs already?

**Comment by trevor-scheer** [comment]
@martinbonnin I think you meant to tag @trevorblades 😄

**Comment by BoD** [comment]
Added a few videos (as animated webp) that I recycled from the announcement blog post :) I'll merge for now, and we can revisit it later if we can add a 'tour' video!
